tsdb/engine/tsm1/engine.go

```diff
@@ -39,9 +39,10 @@ import (
 	"go.uber.org/zap"
 )
 
+//go:generate -command tmpl go run github.com/benbjohnson/tmpl
 //go:generate tmpl -data=@iterator.gen.go.tmpldata iterator.gen.go.tmpl engine.gen.go.tmpl array_cursor.gen.go.tmpl array_cursor_iterator.gen.go.tmpl
-//go:generate go run ../../../_tools/tmpl/main.go -i -data=file_store.gen.go.tmpldata file_store.gen.go.tmpl=file_store.gen.go
-//go:generate go run ../../../_tools/tmpl/main.go -i -d isArray=y -data=file_store.gen.go.tmpldata file_store.gen.go.tmpl=file_store_array.gen.go
+//go:generate go run ../../../internal/cmd/tmpl -i -data=file_store.gen.go.tmpldata file_store.gen.go.tmpl=file_store.gen.go
+//go:generate go run ../../../internal/cmd/tmpl -i -d isArray=y -data=file_store.gen.go.tmpldata file_store.gen.go.tmpl=file_store_array.gen.go
```

Comment:
So some of this code generation needs to happen with the version of `tmpl` in `github.com/benbjohnson/tmpl` and some needs the modified version specific to this repo, I guess.

What are the differences between the two versions, do you know? Can you document this in a comment somewhere?

Reply:
Yes I can. The difference seems to be that the modified tmpl seems to support adding extra data (the `isArray=y`). I think it would be possible to bring this into the mainline `tmpl`, but I didn't want to do that as part of this PR.

Reply:
It looks like this version of `tmpl` was already in the repo at `tools` and I didn't see it so I'll switch to using that one so we don't have multiple copies.